Базовая стилизация всех страниц

Inga-e opened the pull request:
Добавление стилей и корректировка файлов index, catalog

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2166645/tasks/8)

:boom: https://htmlacademy-htmlcss.github.io/2166645-device-35/4/

Inga-e committed:
Базовая стилизация
Добавление стилей и корректировка файлов index, catalog

Inga-e committed:
Меняет базовый цвет у цифр в index

saimon395 commented on styles/styles.css:
```diff
@@ -0,0 +1,424 @@
+.navigation-link,
+.main-navigation-user-text {
+  color: #000000;
+  text-align: center; /*надо ли эту строку и зачем?*/
+  text-decoration: none;
```
нет, не нужно, по крайней мере пока, это не всегда работает, а в дальнейшем возможно по другому центрировать будешь

Inga-e replied:
удалила

saimon395 commented on styles/styles.css:
```diff
@@ -0,0 +1,424 @@
+.basket-set {
+  font-size: 14px;
+  line-height: 17px;
+  background-color: #ffffff;
+  background-size: 34px 34px; /*не устанавливается размер фона как круг, не знаю почему*/
```
потому что размеры самому блоку нужно задать сначала, background-size фон больше самого блока не сделает, плюс это всё равно работает только для картинок

Inga-e replied:
сделала :)

saimon395 commented on styles/styles.css:
```diff
@@ -0,0 +1,424 @@
+  font-size: 18px;
+  line-height: 76px;
+  letter-spacing: 0.2em;
+  text-transform: uppercase;
+  background-color:#ffe17f;
+  background-size: 160px 8px; /*li - строчный элемент, поэтому растягивается. Как исправить?*/
+}
```
просто задать ширину )

Inga-e replied:
Точно! Сделала

saimon395 commented on styles/styles.css:
```diff
@@ -0,0 +1,424 @@
+  letter-spacing: 0.2em;
+  text-transform: uppercase;
+}
+.moving-truck {
+  background-color: #FFE17F;;
+  background-size: 125px 100px;
```
лишнюю точку с запятой удали

Inga-e replied:
Сделала

saimon395 commented on styles/styles.css:
```diff
@@ -0,0 +1,424 @@
+.contacts-address-text,
+.contacts-phone,
+.contacts-schedule-item {
+  font-style: normal;
+  color:#444444;
+  /*нужно убрать наклон*/
```
так его же и так вроде нет

Inga-e replied:
забыла удалить комментарий

Inga-e committed:
Корректирует style, index, catalog

 Внесла корректировки согласно комментариям. Прогнала файлы через валидатор, ошибки исправила.
